_posts/2020-07-13-our-code-review-etiquette.md

```diff
@@ -0,0 +1,78 @@
+- Before submitting your review make sure you have considered <a href="https://www.ombulabs.com/blog/software-development/software-quality/our-definition-of-done.html" target="_blank">Our Definition of "Done"</a>.
+
+#### When you are reviewing
+
+- Understand why the change is necessary (fixes a bug, improves the user experience, refactors the existing code).
+- Be respectful. Keep in mind that there’s a person on the other side of the wire, not a machine, and that it’s hard to understand written words with little context. Avoid letting anger and frustration leak into the review, even if you feel it is justified. There’s no good outcome in those situations.
+- Communicate which ideas you feel strongly about and those you don't.
+- Explain why. Unless the change is about an extremely obvious mistake, explain why you’re suggesting it.
+- Identify ways to simplify the code while still solving the problem.
+- If discussions turn too philosophical or academic, move the discussion to another channel (chat or in email maybe?). In the meantime, let the author make the final decision about the implementation.
+- Avoid trivialities. Think to yourself: all things considered, does it actually matter? Is the cost of the author’s time, and the potential debate, really worth it?
+- Praise the good work. You can praise the logic, design, code organization, or whatever else that you honestly felt was well done.
+- Offer alternative implementations, but assume the author already considered them: <i>"What do you think about using a custom validator here?"</i>
+- When trying to offer alternative solutions, always link to resources if possible to help ensure that the claims being raised are validated.
+- Seek to understand the author's perspective.
+- Read the commit messages if you don't understand something. It's useful to try to understand why the author decided to do something.
+- It’s OK to say it’s all good. That doesn't hurt your reputation or mean that you are a less thoughtful reviewer. If the code is okay, just sign off on the pull request with a 👍 or "Ready to merge" comment.
+
+
+#### When you have your code under review
+
+- Be grateful for the reviewer's suggestions: "Good call. I'll make that change."
+- A common axiom is <i>"Don't take it personally. The review is of the code, not you."</i> We used to include this, but now prefer to say what we mean: Be aware of how hard it is to convey emotion online and how easy it is to misinterpret feedback. If a review seems aggressive or angry or otherwise personal, consider if it is intended to be read that way and ask the person for clarification of intent in private.
+- Write good commit messages. Not just "Added x file", but "Added x class" or "Fixing x bug" if writing a summary, and if it's a complicated/weird bug that took you a while, write a long commit message with references or an explanation.
+- Keeping the previous point in mind: assume the best intention from the reviewer's comments.
```

Comment:
@cleicar This doesn't make sense to me considering the previous point. Maybe you meant the one before that? If so, this might need to be placed in position #3.